src/commands/deploy.ts

```diff
@@ -325,7 +336,11 @@ export async function deployClasses(
             const archivePath = path.join(archivePathTempFolder, `genezioDeploy.zip`);
 
             debugLogger.debug(`Zip the directory ${output.path}.`);
-            await zipDirectory(output.path, archivePath);
+            if (element.language === ".go") {
+                await zipFile(path.join(output.path, "bootstrap"), archivePath);
+            } else {
+                await zipDirectory(output.path, archivePath);
+            }
```

Comment:
I don't like these `if`s that depend on language here. There is also one with `.kt` above these lines. I don't think you should solve this now, but maybe think about a good solution on how to run code based on which language arrives here. We should remove the ts/js bias in our codebase.